### PR TITLE
Added Installation via conda

### PR DESCRIPTION
**Pull request by Yash1256**
Solving #3761 
This is not ready to be merged @ltalirz 
I have added the Installation Instruction via Conda. [Ref](https://gist.github.com/gwangjinkim/f13bf596fefa7db7d31c22efd1627c7a)

Can you tell me the exact position of placing the Installation Instruction??
and there were no installation package for rabbitmq.

### CODE REVIEW

**Comment by ltalirz** [comment]
Hi @Yash1256 , thanks for moving forward with this, and sorry I've not yet proceeded with our previous PR  - we're currently at a plugin/workflows hackathon and very busy ;-)

> Can you tell me the exact position of placing the Installation Instruction??

Actually, putting it at the end is not a bad place to start.
As it gets more tested, we can always move it up.

> and there were no installation package for rabbitmq.

Actually , `conda install -c conda-forge aiida-core.services` should install both postgresql and rabbitmq.

**Comment by Yash1256** [comment]
Sir I can't understand the reason of failing the test cases can you please help me out to proceed??

**Comment by Yash1256** [comment]
@ltalirz I have removed all my changes in the file and then also it is showing 5 failing checks I am now confused whether they were in the inbuilt file or the changes which I have done.
Sir please help me out.

**Comment by ltalirz** [comment]
Dear @Yash1256 - as mentioned previously, there is no need to worry about failing tests at the moment.

Just before you made your first PR, something has changed in our github actions CI testing environment (without changes from our side), and we're working on resolving the issue.
I'll ping you back once it is working properly again and I will properly review your PRs.

Cheers!

**Comment by ltalirz** [comment]
Dear @Yash1256 , we've fixed the CI issue - feel free to put back your changes.

File: docs/source/install/prerequisites.rst [hunk 1]
```diff
@@ -263,3 +263,35 @@ The guide for Ubuntu above can generally be followed, but there are a few things
 It may be worth considering adding some of these commands to your ``~/.bashrc`` file, since some of these settings may reset upon reboot.
 
 .. hint:: For using WSL as a developer, please see the considerations made in our `wiki-page for developers <https://github.com/aiidateam/aiida-core/wiki/Development-environment#using-windows-subsystem-for-linux-wsl>`_.
+
+Installation via CONDA
+========================
+
+To install conda, go to Anaconda `official documentation`_.
```
Comment (ltalirz) [inline docs/source/install/prerequisites.rst]:
```suggestion
To install conda, follow Anaconda's `official documentation`_.
```

File: docs/source/install/prerequisites.rst [hunk 2]
```diff
@@ -263,3 +263,35 @@ The guide for Ubuntu above can generally be followed, but there are a few things
...
+
+.. _official documentation: https://www.anaconda.com/distribution/
+
+If you have not installed the packages yet, you can install them using command
```
Comment (ltalirz) [inline docs/source/install/prerequisites.rst]:
```suggestion
Then, install postgresql and rabbitmq using:
```

File: docs/source/install/prerequisites.rst [hunk 3]
```diff
@@ -263,3 +263,35 @@ The guide for Ubuntu above can generally be followed, but there are a few things
...
+
+.. code-block:: bash
+
+	conda install -c conda-forge aiida-core.services
+
+Now install postgresql via conda
```
Comment (ltalirz) [inline docs/source/install/prerequisites.rst]: this should not be needed, right?

File: docs/source/install/prerequisites.rst [hunk 4]
```diff
@@ -263,3 +263,35 @@ The guide for Ubuntu above can generally be followed, but there are a few things
...
+
+To install conda, go to Anaconda `official documentation`_.
+
+.. _official documentation: https://www.anaconda.com/distribution/
+
+If you have not installed the packages yet, you can install them using command
+
+.. code-block:: bash
+
+	conda install -c conda-forge aiida-core.services
```
Comment (ltalirz) [inline docs/source/install/prerequisites.rst]:
```suggestion
	conda install -y -c conda-forge aiida-core.services
```

File: docs/source/install/prerequisites.rst [hunk 5]
```diff
@@ -263,3 +263,35 @@ The guide for Ubuntu above can generally be followed, but there are a few things
...
+
+Now install postgresql via conda
+
+.. code-block:: bash
+
+	conda install -y -c conda-forge postgresql
+
+Now create a base database locally
+
+..code-block:: bash
```
Comment (ltalirz) [inline docs/source/install/prerequisites.rst]: does this work, i.e. is `initdb` automatically added to the PATH?

File: docs/source/install/prerequisites.rst [hunk 6]
```diff
@@ -263,3 +263,35 @@ The guide for Ubuntu above can generally be followed, but there are a few things
...
+
+	initdb -D mylocal_db
+
+To start the server
+
+..code-block:: bash
+
+	 pg_ctl -D mylocal_db -l logfile start
+
```
Comment (ltalirz) [inline docs/source/install/prerequisites.rst]: you'll also need to start `rabbitmq`, right?

**Comment by ltalirz** [comment]
Could you perhaps address my comments so that we can move forward with this PR?

File: docs/source/install/prerequisites.rst (hunk 5 above)
Comment (Yash1256) [reply docs/source/install/prerequisites.rst]: Yes Sir it works, It automatically added to the path where .conda file is.

**Comment by Yash1256** [comment]
@ltalirz Sorry for coming late to this PR there was some problem over here because of that I couldn't do the required changes. 
Sorry Sir

**Comment by Yash1256** [comment]
@ltalirz Sir any more changes are ti be done or the PR is done ?

File: docs/source/install/prerequisites.rst [hunk 7]
```diff
@@ -263,3 +263,52 @@ The guide for Ubuntu above can generally be followed, but there are a few things
...
+Then, install postgresql and rabbitmq using:
+
+.. code-block:: bash
+
+	conda install -y -c conda-forge aiida-core.services
+
+Now install postgresql via conda
+
+.. code-block:: bash
```
Comment (ltalirz) [inline docs/source/install/prerequisites.rst]: also not needed

File: docs/source/install/prerequisites.rst [hunk 8]
```diff
@@ -263,3 +263,52 @@ The guide for Ubuntu above can generally be followed, but there are a few things
...
+
+Installing `rabbitmq-server` from the `conda-forge` channel can be achieved by adding `conda-forge` to your channels with:
+
+.. code-block:: bash
+
+	conda config --add channels conda-forge
+
+Once the `conda-forge` channel has been enabled, `rabbitmq-server` can be installed with:
+
+.. code-block:: bash
```
Comment (ltalirz) [inline docs/source/install/prerequisites.rst]: also not needed

**Comment by ltalirz** [comment]
Hi @Yash1256 , as mentioned in the first unresolved comment, aiida-core.services already installs rabbitmq and postgresql (at least it should!).
I.e. there is still some cleanup to do in the PR

**Comment by Yash1256** [comment]
@ltalirz if it would we there and we again install then it will show packages already exists, leaving the installation for the rabbitmq command rest all would be same as I think.

**Comment by ltalirz** [comment]
> @ltalirz if it would we there and we again install then it will show packages already exists, leaving the installation for the rabbitmq command rest all would be same as I think.

Hehe, sure - but why would you want to do that?
Installing AiiDA already involves a number of steps; we certainly don't want to make them longer than necessary.

**Comment by ltalirz** [comment]
Closing this PR until change requests are addressed

**Pull request closed without merging**